files/respond/update.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packageManager": "github",
+  "name": "Respond.js",
+  "repo": "scottjehl/Respond",
+  "files": {
+    "basePath": "dest/",
+    "include": [
+      "respond.min.js",
+      "respond.src.js",
+      "respond.matchmedia.addListener.min.js",
+      "respond.matchmedia.addListener.src.js",
+      "../cross-domain/respond-proxy.html",
```

Comment:
One dot is enough here. `./` retains the directory structure. Same for the other files